app/services/symphony_client.rb

```diff
@@ -31,10 +31,13 @@ def login(user_id, password)
     JSON.parse(response.body)
   end
 
-  def authenticate(user)
-    authenticated_request("/user/patron/key/#{user.patron_key}",
-                          headers: { 'x-sirs-sessionToken': user.session_token },
-                          params: { includeFields: 'key' })
+  # This method is for validating user session_token
+  def ping?(user)
+    response = authenticated_request("/user/patron/key/#{user.patron_key}",
+                                     headers: { 'x-sirs-sessionToken': user.session_token },
+                                     params: { includeFields: 'key' })
+
+    response.status == 200
```

Comment:
I'm glad you moved the `response.state == 200` part up here, makes the calling of this method simpler (don't have to do an additional step everytime it's called).